crates/gpui/src/platform/linux/wayland/window.rs

```diff
@@ -306,6 +306,10 @@ impl PlatformWindow for WaylandWindow {
         self.0.inner.borrow_mut().scale
     }
 
+    fn titlebar_top_padding(&self) -> Pixels {
```

Comment:
It seems like this shouldn't be needing the platform for this?

Reply:
We should remove this and call the win32 APIs directly in the platform component.